playbooks/bosh-acceptance-tests/run.yaml

```diff
@@ -0,0 +1,137 @@
+- hosts: all
+  become: yes
+  roles:
+    - role: export-cloud-openrc
+      vars:
+        cloud_name: 'opentelekomcloud'
+  tasks:
+    - name: Run cloudfoundry acceptance tests of bosh-acceptance-tests
```

Comment:
bosh acceptance test is different with cloudfoundry acceptance test, exactly this's should be bosh acceptance tests. So the task name should be "Install bosh and run bosh acceptance tests"

Reply:
done